thenewboston_node/business_logic/tests/test_models/test_field_compaction.py

```diff
@@ -0,0 +1,96 @@
+import pytest
```

Comment:
in addition I would like to see tests that do `to_compact_dict()` and then `from_compact_dict()` and then compare that we got an object equal to original. Because in fact it does not really matter how we compact it (what short field names are), as long as we get the original after uncompacting.

Reply:
https://github.com/thenewboston-developers/thenewboston-node/pull/83/files#diff-638e0785bcd0039bf74e9df592e40ef1fa8573cfd0afe997ff0a1aebbfe073eeR99